README.md

```diff
@@ -36,53 +170,128 @@ F = 1.8 * C + 32
 K = C + 273
 
 a. Make an object called `freezingPoint` that has three properties: `celsius`, `fahrenheit`, and `kelvin`. Give them all values equal to the freezing point of water.
+```
+let freezingPoint = {
+  celsius: 0,
+  fahrenheit: 32,
+  kelvin: 273.2
+}
 
-
+```
 b. Make a class called `Celsius` that has one property: `celsius`, and two methods `getFahrenheitTemp`, and `getKelvinTemp`.
 
-```js
+c. Give `Celsius` a method called `isBelowFreezing` that returns a `Bool` (true if the temperature is below freezing).
+
+```
+class Celsius {
+  constructor(celsius){
+    this.celsius = celsius
+  }
+  getFahrenheitTemp(){
+    return console.log(`It is ${(this.celsius * 1.8) + 32} degrees in Fahrenheit`)
+  }
+  getKelvinTemp() {
+    return console.log(`It is ${this.celsius + 273.15} degrees in Kevlin`)
+  }
+  isBelowFreezing (){
+    if (this.celsius)
+      return false
+  }
+  }
+
 let outsideTempt = new Celsius(10.0)
 outsideTempt.celsius //returns 10.0
 outsideTempt.getKelvinTemp() //returns 283.0
 outsideTempt.getFahrenheitTemp() //returns 50.0
+
 ```
 
-c. Give `Celsius` a method called `isBelowFreezing` that returns a `Bool` (true if the temperature is below freezing).
 
 ## Question 5
 
 a. Create a class called `Movie` that has properties for `name`, `year`, `genre`, `cast`, and `description`. Create an instance of your `Movie`
-
 b. Create an method inside `Movie` called `blurb` that returns a formatted string describing the movie.
 
-Ex: "Borat came out in 2006. It was an odd film starring Sacha Baron Cohen as a man named Borat who was visiting America from Kazakhstan."
+```
+class Movie{
+  constructor(name, year, genre, cast, description){
+    this.name = name;
+    this.year = year;
+    this.genre = genre;
+    this.cast = cast;
+    this.description = description
+  }
+  blurb(){
+    return (`${this.name} came out in ${this.year}. It is a ${this.genre} film starring ${this.cast}. The description of the movie : ${this.description}`)
+  }
+}
+
+let movie1 = new Movie('Soylent green', 1973, 'thriller',['Charlton Heston','Leigh Taylor-Young','Edward G. Robinson']), 'dystopian thriller film')
+
+let movie2 = new Movie('Training Day', 2001, 'thriller',['	
+Denzel Washington','Ethan Hawke','Cop thiller'])
+
+```
+
 
 
 ## Question 6
 
 Write a class Vector that represents a vector in two-dimensional space.
 It takes two number arguments: `x` and `y` parameters, which it should be saved to properties of the same name.
-
+```
+class Vector{
+  constructor(x, y){
+    this.x = x;
+    this.y = y;
+  }
+```
 Give the Vector prototype two methods, `plus` and `minus`, that take another vector as an argument and
 returns a new vector that has the sum or difference of the two vectors’ (the one in `this` and the parameter) x and y values.
+```
+class Vector{
+  constructor(x, y){
+    this.x = x;
+    this.y = y;
+  }
+  plus(v){
+    return console.log(`${this.x + v.x}, ${this.y + v.y}`)
```

Comment:
You want to return a new Vector, and also, you can't return a console log, the logic is good.